### DISCUSSION TITLE
Understanding the Wake-on-LAN (WOL) Feature: Behavior Across LAN, WiFi, and Power States

### DISCUSSION THREAD

**Discussion by amoljagdalepucsd** [Ylianst/MeshCentral#6399] [Q&A]
Hi community ,
                      I am running MeshCentral version 1.1.22 in hybrid mode.I tested the Wake-on-LAN (WOL) feature/action on my systems, which are on the same network. I was able to wake up the device from sleep mode using other live agents on the same network. However, I observed some differences: when the remote system is connected via LAN cable and the power cable is plugged in, it works fine and wakes up from sleep mode. But when the system is connected to the WiFi network or the power cable is unplugged, it does not wake up from sleep mode. I am getting the same behavior on both the localhost server and public server as well.

Is this expected behavior, or do I need to check any settings related to the network or BIOS?

**Comment by si458** [comment]
**you cant wake on lan with WiFi, only LAN cables will work**
to be able to do wake on lan with WiFi, you would need to setup something like intel AMT on your devices,
then configure the AMT to connect over WiFi, 
then you can use the AMT power on instead

**Reply by amoljagdalepucsd** [reply]
Thanks, I got it! So, MeshCentral can’t wake up devices over WiFi without Intel AMT. However, I noticed that even for remote system connected via LAN cable, they need the power cable plugged in for Wake-on-LAN to work. Is this expected behavior?

**Reply by si458** [reply]
yes thats expected behaviour, because they need to be able to stay connected to switches, so the devices need power

**Reply by amoljagdalepucsd** [reply]
okay thanks
